fastlane/Fastfile

```diff
@@ -0,0 +1,152 @@
+# Customise this file, documentation can be found here:
+# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
+# All available actions: https://docs.fastlane.tools/actions
+# can also be listed using the `fastlane actions` command
+
+# Change the syntax highlighting to Ruby
+# All lines starting with a # are ignored when running `fastlane`
+
+# If you want to automatically update fastlane if a new version is available:
+# update_fastlane
+
+# This is the minimum version number required.
+# Update this, if you use features of a newer version
+fastlane_version "2.17.1"
+
+default_platform :ios
+
+platform :ios do
+  before_all do
+    # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
+    ENV["MATCH_KEYCHAIN_NAME"] = "ios-build.keychain"
+    ENV["TEAM_SAGEBIO"] = "4B822CZK9N"
+    ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"] = "KA9Z8R6M6K"
+    ENV["APP_ID_RSDCATALOG"] = "org.sagebase.RSDCatalog"
+    ENV["APP_ID_RSDTEST"] = "org.sagebase.RSDTest"
+  end
+
+  desc "Create keychains to store certificates"
+  lane :keychains do
+    create_keychain(
+      name: ENV["MATCH_KEYCHAIN_NAME"],
+      password: ENV["MATCH_PASSWORD"],
+      default_keychain: true,
+      unlock: true,
+      timeout: 3600,
+      add_to_search_list: true
+    )
+  end
+
+  desc "Fetches provisioning profile and certificates from github repo"
+  lane :certificates do |options|
+    export_method = options[:export_method]
+    match(
+      git_branch: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      type: "development",
+      app_identifier: ENV["APP_ID_RSDCATALOG"],
+      team_id: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
+      keychain_password: ENV["MATCH_PASSWORD"],
+      readonly: true
+    )
+    match(
+      git_branch: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      type: "appstore",
+      app_identifier: ENV["APP_ID_RSDCATALOG"],
+      team_id: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
+      keychain_password: ENV["MATCH_PASSWORD"],
+      readonly: true
+    )
+    match(
+      git_branch: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      type: "development",
+      app_identifier: ENV["APP_ID_RSDTEST"],
+      team_id: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
+      keychain_password: ENV["MATCH_PASSWORD"],
+      readonly: true
+    )
+    match(
+      git_branch: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      type: "appstore",
+      app_identifier: ENV["APP_ID_RSDTEST"],
+      team_id: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
+      keychain_password: ENV["MATCH_PASSWORD"],
+      readonly: true
+    )
+  end
+
+  desc "Execute tests"
+  lane :test do |options|
+    project = options[:project]
```

Comment:
I'm guessing it won't work to point at the project. B/c this is set up using a workspace, the frameworks that are embedded in the workspace are not embedded in the project.

Reply:
Currently pointing to workspace doesn't work because `RSDTestApp` and `RSDCatalog` targets don't exist in ResearchStack2 workspace.   Those targets are in RSDTestApp.xcodeproj. 
 If you want to build targeting the workspace then something needs to be changed in xcode to allow referencing the targets and schemes.

~SageResearch ❯❯❯ xcodebuild -list -project ResearchStack2/ResearchStack2.xcodeproj                                                                                                             
Information about project "ResearchStack2":
    Targets:
        Documentation
        ResearchStack2 (iOS)
        ResearchStack2 (watchOS)
        ResearchStack2 (tvOS)
        ResearchStack2Tests_iOS
        ResearchStack2-UnitTest

    Build Configurations:
        Debug
        Release

    If no build configuration is specified and -scheme is not passed then "Release" is used.

    Schemes:
        Documentation (ResearchStack2)
        ResearchStack2 (iOS)
        ResearchStack2 (tvOS)
        ResearchStack2 (watchOS)